apps/election-manager/src/components/SaveFileToUSB.tsx

```diff
@@ -72,10 +70,11 @@ const SaveFileToUSB = ({
         fileDownload(results, defaultFilename, 'text/csv')
       } else {
         const usbPath = await usbstick.getDevicePath()
+        assert(typeof usbPath !== 'undefined')
```

Comment:
If you are in dev but running with kiosk browser its possible that usbPath is not defined here if you are using the manual save option when there is no usb. The way I wrote this conditional here doesn't make a lot of sense though, added suggestions for a cleaner way. If openFileDialog is true then we're in manual save mode and we don't need to worry about the usb path and if its false then usbPath must be defined.

Reply:
```suggestion
```